src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.PackTool.targets

```diff
@@ -18,7 +18,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     there is no dependency in nuspec. And add Microsoft.NETCore.Platforms, which is used to select correct RID assets.
     -->
   <PropertyGroup>
-    <NETCorePlatformsImplicitPackageVersion Condition=" '$(NETCorePlatformsImplicitPackageVersion)' == '' ">2.0.1</NETCorePlatformsImplicitPackageVersion>
+    <NETCorePlatformsImplicitPackageVersion Condition="'$(NETCorePlatformsImplicitPackageVersion)' == '' and '$(BundledNETCorePlatformsVersion)' != ''">$(BundledNETCorePlatformsVersion)</NETCorePlatformsImplicitPackageVersion>
```

Comment:
Nit: This can be simplified to 

``` xml
<NETCorePlatformsImplicitPackageVersion Condition="'$(NETCorePlatformsImplicitPackageVersion)' == ''">$(BundledNETCorePlatformsVersion)</NETCorePlatformsImplicitPackageVersion>
```

Since we'll still check for empty and fall back to 2.0.1.

Reply:
changed